documentation/source/user_section/tutorials/gm-wm/improving-registration-with-gm-seg/gm-informed-t2s-template-registration.rst

```diff
@@ -0,0 +1,54 @@
+   - ``-initwarp`` : Warping field used to initialize the source image. Here, we supply the ``warp_template2anat.nii.gz`` file from the previous T2 registration. (See: :ref:`template-registration`)
+      - TODO: I'm concerned that the usage of the ``-initwarp`` flag is not clear enough. For users who are following this tutorial standalone, they may not have the full context of the previous tutorial, so the usage of ``-initwarp`` may appear to "come out of nowhere". But, even if they have the context, reusing intermediate results raises some questions:
+          - Is ``-initwarp`` optional or mandatory?
+          - What should the command look like without ``-initwarp``?
+          - Will the registration results be significantly worse without ``-initwarp``?
+          - In what cases should ``-initwarp`` be used, and in what cases can they perform registration directly?
+   - ``-initwarpinv``: Warping field used to initialize the destination image. Here, we supply the inverse warping field, ``warp_anat2template.nii.gz`` from the previous T2 registration. (See: :ref:`template-registration`)
+      - TODO: Same concerns as above.
+   - ``-param`` :
+      - TODO: Why are we using ``rigid`` specifically? The MT tutorial uses ``centermass`` instead... do we need to explain the discrepancy? (Not explained in SCT course.)
+   - ``-qc`` : Directory for Quality Control reporting. QC reports allow us to evaluate the results slice-by-slice.
+
+:Output files/folders:
+   - ``PAM50_t2s_reg.nii.gz`` : The PAM50 template image, registered to the space of the T2s image.
+   - ``t2s_reg.nii.gx``: The T2s image, registered to the space of the PAM50 template.
+   - ``warp_PAM50_t2s2t2s.nii.gz`` : The warping field to transform the PAM50 template to the T2s space.
+   - ``warp_t2s2PAM50_t2s.nii.gz`` : The warping field to transform the T2s image to the PAM50 template space.
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/jn/2857-add-remaining-tutorials/improving-registration-with-gm-seg/io-sct_register_multimodal-t2s.png
+   :align: center
+
+Finally, it is also worth renaming the automatically generated warping fields for clarity.
+
+.. code:: sh
+
+   mv warp_PAM50_t2s2t2s.nii.gz warp_template2t2s.nii.gz
+   mv warp_t2s2PAM50_t2s.nii.gz warp_t2s2template.nii.gz
+
+.. TODO: I've excluded the ``sct_warp_template`` step here because the T2* warped template was not actually used for anything in later steps.
+
+   Also, my intent (more generally) is to convey that the main result of registration is to produce warping fields, and that those warping fields are the thing that should be passed along to other tutorials. (i.e. later in the "metric tutorial" I want to communicate that the user needs a warping field from a previous registration, that way they can warp the template *for the purposes of metric extraction*.)
```

Comment:
> So, I wanted to float the idea that we should only warp the template in a tutorial if the warped template objects are needed (e.g. for metric calculation).

excellent idea. It will be more pedagogical (ie: will force people to think about the _why_ instead of having them mechanically copy/paste all the commands from the tutorial)